### ISSUE TITLE
Creating a Lab, Lab Slug instructions are confusing

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-docs#291]
![image](https://user-images.githubusercontent.com/75502403/214934640-4214f15e-6c54-444a-a469-799a544e67dc.png)

When creating a lab, the instructions use a full URL example when the request is just for the slug at the end of the URL. 

Suggest changing the help text to something like: 

'Enter a valid "slug" (unique text that goes at the end of your lab link) consisting of letters, numbers, underscores, and/or hyphens. For example, inputting "your-lab-name" into the box above will result in "https://lookit.mit.edu/studies/my-lab-name" as your lab's unique URL that will show your discoverable, active studies.'

**Comment by becky-gilbert** [comment]
Excellent point, thanks @ianchandlercampbell!

@mekline are you happy for me to do this? Should I change the default text inside the input field as well?

**Comment by mekline** [comment]
yes and yes!

**Comment by becky-gilbert** [comment]
Hi @ianchandlercampbell, just to confirm: when you say
> When creating a lab, the instructions use a full URL example when the request is just for the slug at the end of the URL.

you're referring to the help text on the form itself (underneath the Custom URL text field), right? Or did this also come up somewhere on the Lookit documentation site? 

Thanks!

**Comment by ianchandlercampbell** [comment]
Yes I am talking about the help text! The current help text makes it appear
as if you should type out the whole URL rather than just the slug

**Comment by becky-gilbert** [comment]
Perfect, thanks @ianchandlercampbell! And I totally agree, just wanted to be sure I didn't miss anything 😄
